### ISSUE TITLE
Recognize Bandcamp links on music pages

### ISSUE THREAD

**Issue by asbjornu** [demozoo/demozoo#379]
Like #143, but for music pages. [Here's a song with a Bandcamp link](https://demozoo.org/music/141390/) that isn't recognized.

**Comment by nswaldman** [comment]
You _really_ shouldn't be offering a pay link as a download source, that's not what the download links are for.

The proper location for this link would be in the external links section.

**Comment by asbjornu** [comment]
True. It was entered into download just out of habit since Demozoo is usually so good at detecting whether the URL is an external site or download. As an external site link, it would still be nice if Demozoo recognized Bandcamp, provided an icon and perhaps even allowed the song to be played through Bandcamp on Demozoo.

**Comment by nswaldman** [comment]
The recognition of the url and providing it with a proper icon is definitely in the scope of the initial request. The latter is a different beast :)
